src/components/Flex/Flex.tsx

```diff
@@ -55,7 +55,7 @@ type Props = {
        * 子要素の間隔。指定しない場合は0
        * xxs=4px, xs=8px, sm=12px, md=16px, lg=24px, xl=40px, xxl=64px
        */
-      spacing: Spacing;
+      spacing?: Spacing;
```

Comment:
When the spacing notation was supported, the absence of spacing was no longer acceptable.
I've changed it back to optional.